doc/articles/uno-development/building-uno-ui.md

```diff
@@ -40,13 +40,12 @@ The step by step process is:
 
 1. Clone the Uno.UI repository locally, and ensure using a short target path, e.g. _D:\uno_ etc.  
 This is due to limitations in the legacy .NET versions used by Xamarin projects. This issue has been addressed in .NET 5, and will come to the rest of the projects in the future.
-1. Make sure you don't have the Uno.UI solution open in any Visual Studio instances. (Visual Studio may crash or behave inconsistently if it's open when the target override is changed.)
+1. Make sure you don't have the Uno.UI solution opened in any Visual Studio instances. (Visual Studio may crash or behave inconsistently if it's open when the target override is changed.)
```

Comment:
@jeromelaban Maybe we can add an additional note here or at the end of the steps to specify that between each platform when changing the identifier in `crosstargeting_override.props` they need again to do the same thing, i.e. closing VS first, updating the identifier, and re-loading the proper solution filter (`.slnf` file) corresponding to the new target platform override they have set

Reply:
@jeromelaban did you have an additional note or step regarding the comment I left? Not sure I'm seeing this info in the updated doc in the PR